tests/Regression/GitHub/3167-class.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+https://github.com/sebastianbergmann/phpunit/pull/3167
+--SKIPIF--
+<?php
+if (defined('HHVM_VERSION')) {
+    print 'skip: PHP runtime required';
```

Comment:
why ?

Reply:
HHVM is not supported.

Reply:
I know, especially why I see no point of putting the condition here.
We don't do it in other tests, right?
Because HHVM is not supported by whole project, not just this one, concrete test.

Like... we don't support PHP4, yet we don't check it in each phpt file ;)

composer is perfect place to specify required engine and version

Reply:
`s/HHVM is not supported/HHVM is not supported, we do not need this check/`

Happy now? :-)

Reply:
haha, yeah, thanks :+1:

Reply:
A+ nitpicking, would buy again